src/ts/vttutils.ts

```diff
@@ -0,0 +1,164 @@
+import { SubtitleRegionContainer, SubtitleLabel } from './components/subtitleoverlay';
+import { VTTProperties, VTTRegionProperties } from 'bitmovin-player/types/subtitles/vtt/API';
+
+// Our default height of a line
+const lineHeight = 28;
+
+enum Direction {
+  Top = 'top',
+  Bottom = 'bottom',
+  Left = 'left',
+  Right = 'right',
+}
+
+const DirectionPair = new Map<Direction, Direction>([
+  [Direction.Top, Direction.Bottom],
+  [Direction.Left, Direction.Right],
+  [Direction.Right, Direction.Left],
+]);
+
+/**
+ * Sets the default standardized styles for the Cue Box
+ * https://w3.org/TR/webvtt1/#applying-css-properties
+ */
+const setDefaultVttStyles = (cueContainer: SubtitleLabel) => {
+  if (cueContainer.vtt.region) {
+    cueContainer.getDomElement().css('position', 'relative');
```

Comment:
I think it would be better to pass the actual DOM Element instead of the `cueContainer` to all the helper functions in `vttutils` as all you ever need from it is `cueContainer.getDomElement()`.

Reply:
The problem is that the type gets lost in that case, the parameter required then would only be `DOM` and not `SubtitleLabel`